<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>890</width>
    <height>675</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QtCharts::QChartView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>411</width>
     <height>261</height>
    </rect>
   </property>
  </widget>
  <widget class="QtCharts::QChartView" name="graphicsView_2">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>60</y>
     <width>411</width>
     <height>261</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>330</y>
     <width>121</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>随机读操作性能测试</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>10</y>
     <width>236</width>
     <height>36</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>130</width>
     <height>18</height>
    </size>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;HMFS性能测试与对比&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>330</y>
     <width>121</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>随机写操作性能测试</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>330</y>
     <width>231</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar_2">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>330</y>
     <width>231</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QtCharts::QChartView" name="graphicsView_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>370</y>
     <width>411</width>
     <height>261</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>640</y>
     <width>121</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>顺序读操作性能测试</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar_3">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>640</y>
     <width>231</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QtCharts::QChartView" name="graphicsView_4">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>370</y>
     <width>411</width>
     <height>261</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>640</y>
     <width>121</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>顺序写操作性能测试</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar_4">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>640</y>
     <width>231</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QtCharts::QChartView</class>
   <extends>QGraphicsView</extends>
   <header location="global">qchartview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
